examples/DotNetCore/WebDemoWithEventHub/WebDemoWithEventHub/EventHubService.cs

```diff
@@ -74,9 +76,14 @@ private Task ProcessEventHandler(ProcessEventArgs eventArgs)
         {
             _logger.LogInformation("EventHub update received. Triggering cache invalidation.");
 
-            //
-            // Set the cached value for key-values registered for refresh as dirty.
-            _configurationRefresher?.SetDirty();
+            // Build EventGridEvent from notification message
+            EventGridEvent eventGridEvent = EventGridEvent.Parse(BinaryData.FromBytes(eventArgs.Data.EventBody));
```

Comment:
@MBSolomon can you please confirm this is the right code to convert Event Hubs event to Event Grid event?

Reply:
Here do we need to validate if EventArgs.Data is null? In the documentation it is expected to be null if the receive call times out. So this may cause issues if we try and pull the EventBody from a null value. https://docs.microsoft.com/en-us/dotnet/api/azure.messaging.eventhubs.processor.processeventargs?msclkid=bbd79e46cfb011ecba338c0ae577e165&view=azure-dotnet

Reply:
Updated. Thanks!